__tests__/app.test.js

```diff
@@ -61,21 +61,38 @@ describe("GET /api/articles/", () => {
   });
   test("status 200 responds with articles with article_id only and returns comment_count as well", () => {
     return request(app)
-      .get("/api/articles/3")
+      .get("/api/articles/4")
       .expect(200)
       .then(({ body }) => {
-        expect(body.article).toEqual({
-          article_id: 4,
-          title: "Student SUES Mitch!",
-          topic: "mitch",
-          author: "rogersop",
-          body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
-          created_at: "2020-05-06T01:14:00.000Z",
-          votes: 0,
-          comment_count: expect.any(String),
+        expect(body.article).toEqual({title: "Student SUES Mitch!",
+        article_id: 4,
+        topic: "mitch",
+        author: "rogersop",
+        body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+        created_at: expect.any(String),
+        votes: 0,
+        comment_count: expect.any(String)
         });
       });
   });
+  test.only("status 200 responds with articles that match the topic if a topic is sent", ()=> {
```

Comment:
don't forget to remove .onlys and run all the tests to make sure everything works!